Moya.xcodeproj/project.pbxproj

```diff
@@ -302,23 +288,20 @@
 			isa = PBXHeadersBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				5B9281C0308BF58977D27DE9 /* ReactiveMoya.h in Headers */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
 		9F6005B4252938375E1B61C9 /* Headers */ = {
 			isa = PBXHeadersBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				55D38822179474648606A08E /* RxMoya.h in Headers */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
 		F4CA905CBCCD3A2D1D028529 /* Headers */ = {
 			isa = PBXHeadersBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				42414D9A4174BE3C8A63A2D4 /* Moya.h in Headers */,
```

Comment:
I think if you click on the `.h` files and go to _File Inspector_ -> Check the respective box for Target Membership this should resolve the issue:
![screen shot 2017-07-05 at 3 33 05 pm](https://user-images.githubusercontent.com/7445580/27883784-65512484-6197-11e7-8135-c08763a969af.png)